src/components/tooltip/tooltip.e2e.ts

```diff
@@ -561,6 +561,35 @@ describe("calcite-tooltip", () => {
     expect(await tooltip.getProperty("open")).toBe(false);
   });
 
+  it("should close tooltip when closeOnClick is true and referenceElement is clicked quickly", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      html`
+        <calcite-tooltip reference-element="ref" close-on-click>Content</calcite-tooltip>
+        <button id="ref">Button</button>
+      `
+    );
+
+    await page.waitForChanges();
```

Comment:
Nitpick: Can you confirm if this is needed here? I was under the impression that `setContent` called this internally, but there's [no mention of this in the doc](https://stenciljs.com/docs/end-to-end-testing#:~:text=setContent(html%3A%20string)). Worth noting that this behavior is documented in the [spec page doc](https://stenciljs.com/docs/unit-testing#:~:text=return%20null.-,setContent(html),-Convenience%20function%20to).